tasks/generate.yml

```diff
@@ -1,8 +1,10 @@
 ---
- - name: Generate RSA key
-   command: openssl genrsa -out {{ssl_certs_privkey_path}} {{ssl_certs_key_size}} creates={{ssl_certs_privkey_path}}
- - name: Generate CSR
-   command: openssl req -new -subj '{{ssl_certs_fields}}' -key {{ssl_certs_privkey_path}} -out {{ssl_certs_csr_path}} creates={{ssl_certs_csr_path}}
- - name: Generate self-signed SSL certificate
-   command: openssl req -nodes -x509 -days {{ssl_certs_days}} -in {{ssl_certs_csr_path}} -key {{ssl_certs_privkey_path}} -out {{ssl_certs_cert_path}} -extensions v3_ca creates={{ssl_certs_cert_path}}
-   when: ssl_certs_generate_self_signed
\ No newline at end of file
+  - name: Generate RSA key
+    command: openssl genrsa -out {{ ssl_certs_privkey_path }} {{ ssl_certs_key_size }} creates={{ ssl_certs_privkey_path }}
+
+  - name: Generate CSR
+    command: openssl req -new -subj "{{ ssl_certs_fields }}" -key {{ ssl_certs_privkey_path }} -out {{ ssl_certs_csr_path }} creates={{ ssl_certs_csr_path }}
```

Comment:
I used extra_vars because ansible_fqdn was still grabbing the too long
hostname. I think that might be because travis-ci has switched to using
Docker.

I changed the quotes to match the quotes used elsewhere, but you are right,
it could impact the shell command, tho in this case I don't think it does.

On Tue, Dec 29, 2015, 5:27 AM Julien DAUPHANT <notifications@github.com>
wrote:

> In tasks/generate.yml
> <https://github.com/jdauphant/ansible-role-ssl-certs/pull/3#discussion_r48539974>
> :
>
> > \ No newline at end of file
> > +  - name: Generate RSA key
> > +    command: openssl genrsa -out {{ ssl_certs_privkey_path }} {{ ssl_certs_key_size }} creates={{ ssl_certs_privkey_path }}
> > +
> > +  - name: Generate CSR
> > +    command: openssl req -new -subj "{{ ssl_certs_fields }}" -key {{ ssl_certs_privkey_path }} -out {{ ssl_certs_csr_path }} creates={{ ssl_certs_csr_path }}
>
> Change Quote here, have an impact on the Shell command
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdauphant/ansible-role-ssl-certs/pull/3/files#r48539974>
> .
>